src/main/kotlin/no/nav/helse/soknad/Søknad.kt

```diff
@@ -191,11 +193,27 @@ data class Ferieuttak(
 data class Frilans(
     @JsonFormat(pattern = "yyyy-MM-dd")
     val startdato: LocalDate,
-    val jobberFortsattSomFrilans: Boolean
+    val jobberFortsattSomFrilans: Boolean,
+    val arbeidsforhold: Arbeidsforhold? = null
 )
 
 enum class Årsak {
     BARNET_INNLAGT_I_HELSEINSTITUSJON_FOR_NORSK_OFFENTLIG_REGNING,
     BARNET_INNLAGT_I_HELSEINSTITUSJON_DEKKET_ETTER_AVTALE_MED_ET_ANNET_LAND_OM_TRYGD,
     ANNET,
 }
+
+data class Arbeidsforhold(
+    val skalJobbe:SkalJobbe,
+    val arbeidsform: Arbeidsform,
+    val jobberNormaltTimer: Double,
+    val skalJobbeTimer: Double,
+    val skalJobbeProsent: Double
+)
+
+enum class SkalJobbe(@JsonValue val verdi: String) {
```

Comment:
Hvorfor kan ikke frontend bare sende verdiene som Enum, sånn som det gjøres for Arbeidsform. ? Altså JA, NEI, REDUSTERT, VET_IKKE. Så blir det likt og vi slipper å bruke @JsonValue

Reply:
Det hadde vært bedre ja. Skal høre med Aleksei.

Reply:
Se siste commit.